### PR TITLE
feature, add seed input for regenerating the same list every time

### PR DESCRIPTION
**Pull request by dsgepers**
I'm not generally a frontender but I've posted the suggestion for this on Reddit, and figured I might as well make a PR out of it.

Code might not be up to par cause im a sucky JavaScripter, be gentle :-)

### CODE REVIEW

File: src/journey.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import { all, assoc, equals, findLast, pipe, reject } from "ramda";
 import { v4 } from "uuid";
+var seedrandom = require('seedrandom');
```
Comment (kwhitaker) [inline src/journey.js]: This should be a `const`. Also, can we use the `import` syntax here, instead of the `require` ?

File: package.json [hunk 2]
```diff
@@ -8,6 +8,7 @@
     "preact": "8.2.5",
     "preact-compat": "3.17.0",
     "ramda": "0.25.0",
+    "seedrandom": "^2.4.3",
```
Comment (kwhitaker) [inline package.json]: we can go ahead and lock this down (remove the `^`).

File: src/journey.js [hunk 3]
```diff
@@ -32,6 +33,7 @@ const genWandering = () => [0, 1].map(() => toDirection[d6()]);
 const genEncounter = () => (d20() >= 16 ? d100() : "none");
 
 const getWeather = day => assoc("weather", genWeather(), day);
+export const setSeed = seed => { seedrandom(seed, {global: true}); }
```
Comment (kwhitaker) [inline src/journey.js]: Would you mind putting a comment above this line, indicating that `global: true` is going to affect the `Math` prototype? At least, I _think_ that's what is happening, based on the documentation.

**Review by kwhitaker — CHANGES_REQUESTED** [review]
Overall this looks good. I've just a got a few minor changes I'd appreciate.

Thanks so much!

File: package.json (hunk 2 above)
Comment (dsgepers) [reply package.json]: Changed this in: 66fbdb0

File: src/journey.js (hunk 1 above)
Comment (dsgepers) [reply src/journey.js]: I was actually trying to do this earlier, figured it out now though: Changed this in: 66fbdb0

File: src/journey.js (hunk 3 above)
Comment (dsgepers) [reply src/journey.js]: Changed this in: 66fbdb0

**Review by kwhitaker — APPROVED** [review]

**Comment by kwhitaker** [comment]
Merged! I'll get it deployed ASAP

**Comment by dsgepers** [comment]
Awesome! 👍

**Pull request merged**